dotcom-rendering/src/web/lib/renderElement.tsx

```diff
@@ -514,13 +514,7 @@ export const renderElement = ({
 		case 'model.dotcomrendering.pageElements.SpotifyBlockElement':
 			return [
 				true,
-				<ClickToView
-					role={element.role}
-					isTracking={element.isThirdPartyTracking}
-					isMainMedia={isMainMedia}
-					source={element.source}
-					sourceDomain={element.sourceDomain}
-				>
+				<Island deferUntil="visible">
```

Comment:
Whats the reason for moving ClickToView into `SpotifyBlockContent`? Is it suggesting that we always click to view the `SpotifyBlockContent` and blocks like it?

Reply:
Yes, basically it is not only technically required now but, as [Olly pointed out on another PR](https://github.com/guardian/dotcom-rendering/pull/3824#issuecomment-1011022614) the other day, it also helps firm up the legal requirement that it must always be used. With it integral to the component, you can't accidentally forget it.

Reply:
> the legal requirement that it must always be used

Great reason.